napari/layers/points/_tests/test_points_mouse_bindings.py

```diff
@@ -51,7 +51,7 @@ def create_known_points_layer_2d():
         testing when needing to guarantee no point is clicked on.
     """
     data = [[1, 3], [8, 4], [10, 10], [15, 4]]
-    known_non_point = [20, 30]
+    known_non_point = [10, 11]
```

Comment:
Why is your known non-point different?

Reply:
Update: ah, I don't think that's important, never mind.

Reply:
Well, for future reference: The point used to be far "outside" the other points bounding box. I moved it in between (and close to one existing point) to ensure that almost-overlappin gpoints would work as intended.